src/styles/our-mission.scss

```diff
@@ -0,0 +1,9 @@
+@import "_design-system.scss";
+
+.jf-text-block-with-spacing {
+  p {
```

Comment:
does this paragraph style apply to the visionContent.json element? could we just wrap it in a `<p>` tag instead?

Reply:
yes, that was the idea. Good point about making the safer/more explicit with the <p> tag. I've added that.

Reply:
oops sorry i meant h4! like, if it was `<h4>{documentToReactComponents(props.content.visionContent.json)}</h4>` would that remove the need for this custom css? or is that a no-go because it's automatically wrapped in the paragraph element?

Reply:
yeah exactly, documentToReactComponents always wraps it in `<p>` here. But I just updated this after seeing how this was done for the articles using the contentful `BLOCKS`, so I think this is the best way to handle it.